core/video_options.hpp

```diff
@@ -13,6 +13,68 @@
 
 #include "options.hpp"
 
+struct Bitrate
+{
+public:
+	Bitrate() : bps_(0) {}
+
+	void set(const std::string &s)
+	{
+		static const std::map<std::string, uint64_t> match
+		{
+			{ "bps", 1 },
+			{ "b", 1 },
+			{ "kbps", 1000 },
+			{ "k", 1000 },
+			{ "mbps", 1000 * 1000 },
```

Comment:
Wait for the comment over supporting milli bits/sec.... ;-)

Reply:
Haha, there was a whole conversation about it this morning :)